trackbasket_be/app.py

```diff
@@ -11,21 +11,21 @@
 from models.store import Store
 from resources.items import Items
 from resources.list_shopping_lists import ListShoppingLists
-
-POSTGRES = {
-    'user': 'postgres',
-    'pw': 'password',
-    'db': 'my_database',
-    'host': 'localhost',
-    'port': '5432',
-}
+import os
+# POSTGRES = {
+#     'user': 'postgres',
+#     'pw': 'password',
+#     'db': 'my_database',
+#     'host': 'localhost',
+#     'port': '5432',
+# }
 
 app = Flask(__name__)
 api = Api(app)
 
 
-
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://%(user)s:\%(pw)s@%(host)s:%(port)s/%(db)s' % POSTGRES
+app.config['SQLALCHEMY_DATABASE_URI'] = os.environ['DATABASE_URL']
```

Comment:
please run 
```
export DATABASE_URL="postgresql://postgres:\password@localhost:5432/my_database"
```
from the folder of the application while you are in the virtual environment

Reply:
looking good!